fandor/lambdaFunction
# Lambda edits

#### Purpose of Pull Request?
To send user information to iterable every time update_user_types is called.
#### Provide link to Tracker Story:
#### How should this code be validated?
Add a file to the s3 bucket "fandor reach engine integration" ( for testing/proof of concept, this will be updated later. Verify that the sns topic "update film" has the file
#### Any background context you want to provide?
#### Questions:
#### Extra Release Steps:
